compiler/src/dotty/tools/dotc/transform/InstrumentCoverage.scala

```diff
@@ -273,24 +273,34 @@ class InstrumentCoverage extends MacroTransform with IdentityDenotTransformer:
      * should not be changed to {val $x = f(); T($x)}(1) but to {val $x = f(); val $y = 1; T($x)($y)}
      */
     private def needsLift(tree: Apply)(using Context): Boolean =
-      def isBooleanOperator(fun: Tree) =
-        // We don't want to lift a || getB(), to avoid calling getB if a is true.
-        // Same idea with a && getB(): if a is false, getB shouldn't be called.
-        val sym = fun.symbol
-        sym.exists &&
+      inline def isShortCircuitedOp(sym: Symbol) =
```

Comment:
The JVM should already do a good job at inlining short methods, so I don't think `inline` is useful here or in the next method.